_posts/2021-02-18-survey/survey.Rmd

```diff
@@ -0,0 +1,314 @@
+
+Last week, [torch, tidymodels, and high-energy physics](https://blogs.rstudio.com/ai/posts/2021-02-11-tabnet/) featured the first `tidymodels`-integrated `torch` package. And there's more to come. In fact, if you are developing a package in the `torch` ecosystem, why not consider doing the same? Should you run into problems, the growing `torch` community will be happy to help.
+
+##### Documentation, examples, teaching materials
+
+Thirdly, several respondents expressed the wish for more documentation, examples, and teaching materials. Let me say that here, the situation is different for TensorFlow than for `torch`.
+
+For [`tensorflow`](https://tensorflow.rstudio.com/), the website has a multitude of guides, tutorials, and examples. For [`torch`](torch.mlverse.org), reflecting the discrepancy in respective lifecycles, materials are not that abundant (yet). However, after a recent refactoring, the website has a new, four-part [Get started](https://torch.mlverse.org/start/) section addressed to both beginners in DL and experienced TensorFlow users curious to learn about `torch`. After this hands-on introduction, a good place to get more technical background would be the section on [tensors, autograd, and neural network modules](https://torch.mlverse.org/technical/).
+
+Truth be told, though, nothing would be more helpful here than contributions from the community. Whenever you solve even the tiniest (which is often how things appear to oneself) problem, consider creating a vignette explaining what you did. Future users will be thankful, and a growing user base means that over time, it'll be your turn to find that some things have already been solved for you!
+
+Which leads me to the last -- aggregate -- point.
+
+##### Community, community, community
+
+The remaining things I'd like to mention didn't come up quite as often (individually), but taken together, they all have something in common: They all are wishes *we* happen to have, as well!
+
+This definitely holds in the abstract -- let me cite:
+
+> "Develop more of a DL community"
+>
+> "Larger developer community and ecosystem. Rstudio has made great tools, but for applied work is has been hard to work against the momentum of working in Python."
+
+We wholeheartedly agree: And building a larger community is exactly what we're trying to do. I like the formulation "a DL community" insofar it is framework-independent. In the end, frameworks are just tools, and what counts is our ability to usefully apply those tools to problems we need to solve.
+
+Concrete wishes include
+
+-   More paper/model implementations (such as TabNet).
+
+-   Facilities for easy data reshaping and pre-processing (e.g., in order to pass data to RNNs or 1dd convnets in the expected 3-d format).
+
+-   Probabilistic programming for `torch` (analogously to TensorFlow Probability)
```

Comment:
```suggestion
-   Probabilistic programming for `torch` (analogously to TensorFlow Probability).
```

Reply:
Always check for list-agreement/consistency with punctuation at the end of each bullet! 🤓

Reply:
lol right